Library/Formula/libqxt.rb

```diff
@@ -0,0 +1,233 @@
+class Libqxt < Formula
+  desc "Extension library for Qt providing a suite of utility classes"
+  homepage "https://bitbucket.org/libqxt/libqxt/wiki/Home"
+
+  stable do
+    url "https://bitbucket.org/libqxt/libqxt/get/v0.6.2.tar.gz"
+    sha256 "582426d6c81958dd1ac5ca4a9d49807e0a2204b7343e7d49c6613bcb148b5cb8"
+    patch :DATA
```

Comment:
Please link to where upstream this has been submitted/etc, Thanks. It may be better to submit this to https://github.com/Homebrew/patches and then use that link in a `patch do` blob. That would allow you to use `patch :DATA` for the `head` stage if necessary without writing it inline.